CesiumGltf/generated/include/CesiumGltf/AccessorSpec.h

```diff
@@ -146,6 +154,25 @@ struct CESIUMGLTF_API AccessorSpec : public CesiumGltf::NamedObject {
    */
   std::optional<CesiumGltf::AccessorSparse> sparse;
 
+  int64_t getSizeBytes() const {
+    int64_t accum = 0;
+    accum += sizeof(this->bufferView);
+    accum += sizeof(this->byteOffset);
+    accum += sizeof(this->componentType);
+    accum += sizeof(this->normalized);
+    accum += sizeof(this->count);
+    accum += sizeof(this->type);
+    for (const double& value : this->max) {
+      accum += sizeof(value);
+    }
```

Comment:
Can we add some custom logic for arrays of primitives, so that this is generated as `accum += sizeof(double) * this->max.capacity()`?